docs/LOCAL_DEPLOYMENT.md

```diff
@@ -0,0 +1,67 @@
+
+Local Deployment
+================
+### Overview
+This document and other documents, referenced here, contain steps to git clone the latest repository, deploying the microservices locally and also a walk through the general repository and microservice structure.
+&nbsp; 
+### Steps to clone the restructured repository
+#### If you have an existing cloned repository
+1) Use recurssive clone to clone the repository and the sub-modules.
+````console
+#### For git version 2.17.1
+abc@pcs git clone --recurse-submodules  https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet.git
+Cloning into 'azure-iot-pcs-remote-monitoring-dotnet'...
+remote: Counting objects: 774, done.
+.....
+Submodule 'cli' (https://github.com/Azure/pcs-cli.git) registered for path 'cli'
+Submodule 'services' (https://github.com/Azure/remote-monitoring-services-dotnet.git) registered for path 'cli'
+Submodule 'webui' (https://github.com/Azure/azure-iot-pcs-remote-monitoring-webui.git) registered for path 'webui'
+.....
+````
+2) It is safe to delete exisiting microservice folders (except for services folder) under the top-level of the repository. 
```

Comment:
it might be good to specify that webui and cli should not be deleted

Reply:
done